walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,83 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
```

Comment:
Prefer including small imports. Do you need everything in java.util?

Reply:
Nope, I can break it up. Done in 867219a19e5eefd94551c1ab464e0e388654d5e0.